Header header

# latitude (in degrees), + to N, - to S
float64 latitude

# latitude (in degrees), + to E, - to W
float64 longitude